src/pipelines/epidemiology/gb_authority.py

```diff
@@ -50,12 +91,46 @@ def parse_dataframes(
 
 
 class Covid19UkL1DataSource(Covid19UkL2DataSource):
-    def parse_dataframes(
-        self, dataframes: Dict[str, DataFrame], aux: Dict[str, DataFrame], **parse_opts
-    ) -> DataFrame:
-        data = super().parse_dataframes(dataframes, aux, **parse_opts)
+    def parse(self, sources: Dict[str, str], aux: Dict[str, DataFrame], **parse_opts) -> DataFrame:
+        data = super().parse(sources, aux, **parse_opts)
 
         # Aggregate data to the country level
         data = data.groupby("date").sum().reset_index()
         data["key"] = "GB"
         return data
+
+
+class Covid19UkL3DataSource(DataSource):
+    """ This API can be used to get historical case data for utla and ltla in the UK """
+
+    def parse(self, sources: Dict[str, str], aux: Dict[str, DataFrame], **parse_opts) -> DataFrame:
+        cases = {
```

Comment:
I suppose deaths data is not available for L3? How about tests and hospitalizations?

Reply:
Yeah no deaths data, will investigate the others once this is through. I focused on cases, because it reproduces what we previously had.